src/hokusai/hokusai.yml

```diff
@@ -24,9 +24,14 @@ commands:
       - checkout
 
   setup-docker:
+    parameters:
+      remote_docker_version:
+        type: string
+        default: 19.03.13
     steps:
       - setup
-      - setup_remote_docker
+      - setup_remote_docker:
+          version: << parameters.remote_docker_version >>
```

Comment:
Nice a parameter works here - was not sure if it would be valid for interpolation